test/uk/gov/hmrc/merchandiseinbaggagefrontend/pagespecs/GoodsRouteDestinationPageSpec.scala

```diff
@@ -45,5 +45,5 @@ class GoodsRouteDestinationPageSpec extends DeclarationDataCapturePageSpec[YesNo
   // PH - I think this should be persisted on the DeclarationJourney
   // and we should check the user has answered the question before /check-your-answers
   // otherwise the user could force browse and avoid answering the question, or submit a declaration for an invalid case
-  override def extractFormDataFrom(declarationJourney: DeclarationJourney): Option[YesNo] = Some(Yes)
+  override def extractFormDataFrom(declarationJourney: DeclarationJourney): Option[YesNo] = declarationJourney.maybeGoodsRouteDestination
 }
```

Comment:
I think you should add some tests in here:

https://github.com/hmrc/merchandise-in-baggage-frontend/blob/a90857b04230f061caf5cee695b1e8fccc2d92c7/test/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/DeclarationSpec.scala#L262

to show that the declaration is incomplete if the destination is NI and the goods are moving to/from the EU